crates/sncast/src/helpers/token.rs

```diff
@@ -0,0 +1,29 @@
+use clap::ValueEnum;
+use serde::Serialize;
+use starknet::macros::felt;
+use starknet_types_core::felt::Felt;
+
+const STRK_CONTRACT_ADDRESS: Felt =
+    felt!("0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d");
+
+const ETH_CONTRACT_ADDRESS: Felt =
+    felt!("0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7");
+
+#[derive(Default, Serialize, Clone, Copy, Debug, ValueEnum, strum_macros::Display)]
+#[strum(serialize_all = "lowercase")]
+#[serde(rename_all = "lowercase")]
```

Comment:
Let's add a comment in code about why we need them both so someone doesn't remove it by accident.